minio/mc
# Skip subnet config get while `--airgap` mode of license register

## Description
While running the `mc license register ALIAS/ --airgap` we can avoid getting subnet config details as its not mandatory to verify if SUBNET is actually accessible. This is in link with backard compatibility of `mc` as well where older version of minio may not support subsystem `subnet` itself.

## Motivation and Context
User should be able to get registration URL in airgap mode with older versions of minio as well where `subnet` is not a valid subsystem. This way he/she will be able to register the instance and get required support for the instance.

## How to test this PR?
Follow below steps for verification of the changes

Step-1: Start an older version (e.g. ELEASE.2021-04-22T15-44-28Z) of minio
```
✘ ./minio server ./data/1p1host{1...4}/disk{1...4} --address :9010
```
Step-2: Create alias for the new instance
```
✘ ./mc alias set myminio http://localhost:9010 minioadmin minioadmin
Added `myminio` successfully.
```
Step-3: Try to register the new instance in airgap mode
```
✘ ./mc license register myminio --airgap --dev 
Open the following URL in the browser to register myminio on SUBNET:
http://localhost:9000/cluster/register?token=eyJkZXBsb3ltZW50X2lkIjoiM2FhM2NlZjEtNTAwNi00ODg4LWJkN2EtYTJhNzYwYmFiNzdjIiwiY2x1c3Rlcl9uYW1lIjoibXltaW5pbyIsInVzZWRfY2FwYWNpdHkiOjAsImluZm8iOnsibWluaW9fdmVyc2lvbiI6IjIwMjMtMDItMjFUMTM6MDQ6MzdaIiwibm9fb2Zfc2VydmVyX3Bvb2xzIjoxLCJub19vZl9zZXJ2ZXJzIjoxLCJub19vZl9kcml2ZXMiOjE2LCJub19vZl9idWNrZXRzIjowLCJub19vZl9vYmplY3RzIjowLCJ0b3RhbF9kcml2ZV9zcGFjZSI6ODA5MTAwNjE0MDQxNiwidXNlZF9kcml2ZV9zcGFjZSI6MTIxNzM2MjAwMTkyMH19
```
Step-4: Open above URL and login with SUBNET credentials. You should be able to see the registered instance

![Screenshot from 2023-02-21 20-50-26](https://user-images.githubusercontent.com/6771252/220385513-1848460f-a859-483e-9031-2248b36f4166.png)

Step-5: Download the license from the SUBNET portal and try updating license as below
```
✘ ./mc license update myminio ./license.key --dev
License updated successfully for myminio
```

## Types of changes

## Checklist:
